src/libutil/experimental-features.cc

```diff
@@ -152,13 +152,6 @@ constexpr std::array<ExperimentalFeatureDetails, 14> xpFeatureDetails = {{
             may confuse external tooling.
         )",
     },
-    {
-        .tag = Xp::FetchClosure,
-        .name = "fetch-closure",
-        .description = R"(
-            Enable the use of the [`fetchClosure`](@docroot@/language/builtins.md#builtins-fetchClosure) built-in function in the Nix language.
-        )",
-    },
```

Comment:
Is this the process for stabilizing a feature? I think we should silently ignore it until well after, because they're harmless and that way users can use mixed Nix versions with the same config without getting annoyed with non actionable warnings.

I guess we did it this way in https://github.com/NixOS/nix/pull/8322

Maybe we could have a list of ignored features so that we don't warn for those?

Reply:
I don't think we did it in #8322, but that seems to be a great idea.
I think we can just keep the experimental flag and mark it as deprecated with no effect. Or have an extra field specifying that it should be enabled by default